intro-to-ruby.rb

```diff
@@ -1,14 +1,20 @@
 #create a string and store it in a variable
+food = "chicken,french fries,pizza"
+puts " like to eat #{food} after school."
 
 #print out the value of the variable you just created
 
+
 #print the sum of 185 and 14
+puts (185+14)
 
 #print the product of 1.8 and 3.6
-
+puts (1.8*3.6)
 #store the number 78 as a variable
+x="78"
 
 #print the average of the following numbers: 78, 55, 63, 98
+puts add 78,55,63,98
```

Comment:
Ruby accepts math just as you did in line 9. How could you change it so that you find the average of these numbers?